Gruntfile.js

```diff
@@ -265,7 +218,10 @@ var _              = require('lodash'),
                 coverage: {
                     // TODO fix the timing/async & cleanup issues with the route and integration tests so that
                     // they can also have coverage generated for them & the order doesn't matter
```

Comment:
This todo can also be dropped. Everything can be run together in any order as of #6670.